src/analyses/replace_symbol_ext.cpp

```diff
@@ -38,20 +38,7 @@ bool replace_symbol_extt::replace(exprt &dest) const
   if(!have_to_replace(dest))
     return result;
 
-  // do not replace object in address_of expressions
-  if(dest.id()==ID_address_of)
-  {
-    const exprt &object = to_address_of_expr(dest).object();
-    if(object.id()==ID_symbol)
-    {
-      expr_mapt::const_iterator it=
-        expr_map.find(object.get(ID_identifier));
-
-      if(it!=expr_map.end())
-        return false;
-    }
-  }
-  else if(dest.id()==ID_symbol)
+  if(dest.id()==ID_symbol)
```

Comment:
Or I think we can just generally don't replace symbols in address of expressions with constant expressions. It seems that this would never be meaningful.